vue-js/README.md

```diff
@@ -1,5 +1,20 @@
 # Exercise \#1
 
+## Usage
+You need nodejs to execute this project!
+First of all, you have to install vue cli:
```

Comment:
No you don't.  Package managers like `npm` and `yarn` can resolve binaries in `node_modules/.bin`. So you can run a binary in a script in `package.json` and your package manager will lookup the local binary folder first before using `$PATH` as fallback.